phpstan-baseline.neon

```diff
@@ -2732,6 +2737,16 @@ parameters:
 			count: 2
 			path: app/Http/Controllers/SubProjectController.php
 
+		-
+			message: "#^Access to an undefined property object\\:\\:\\$schema_file\\.$#"
+			count: 1
+			path: app/Http/Controllers/SubmissionController.php
+
+		-
+			message: "#^Call to an undefined method object\\:\\:validate_xml\\(\\)\\.$#"
+			count: 1
+			path: app/Http/Controllers/SubmissionController.php
```

Comment:
These seem like real issues.  Instead of having a `validate_xml()` method only on the abstract XML handler, what do you think about adding a generic `validate()` method on the abstract handler class with a dummy implementation, and then overriding it in the abstract XML handler?